.github/workflows/build.yml

```diff
@@ -551,6 +564,7 @@ jobs:
     - check_generated_files
     - build_macos
     - build_macos_free_threading
+    - build_macos_free_threading_with_stackref_debug
```

Comment:
Could we do this for normal builds as well?
The linux machines are cheapest, so we should probably use those.

Reply:
We can't because normal builds require all the inline functions to be macros https://github.com/python/cpython/issues/121263. And these are too complicated to fit into a single macro.